autotest/gdrivers/jpegxl.py

```diff
@@ -70,9 +70,9 @@ def test_jpegxl_rgba():
 
 
 @pytest.mark.parametrize("lossless", ["YES", "NO", None])
-def test_jpegxl_rgba_lossless_param(lossless):
+def test_jpegxl_rgba_lossless_param(tmp_vsimem, lossless):
     src_ds = gdal.Open("../gcore/data/stefan_full_rgba.tif")
-    filename = "/vsimem/jpegxl_rgba_lossless_param.jxl"
+    filename = str(tmp_vsimem / "jpegxl_rgba_lossless_param.jxl")
```

Comment:
Harmless, and I wouldn't bother changing it, but there should be no need for `str` here? Or is something still lacking support for `os.PathLike` ?

Reply:
you're right. I've removed str() usage in ``str(tmp_vsimem / "xxx")``.  The only places where I needed to use str() is when appending a .aux.xml suffix